tests/Microsoft.Identity.Test.Unit.net45/ApiConfigTests/AcquireTokenInteractiveBuilderTests.cs

```diff
@@ -119,5 +120,18 @@ await AcquireTokenInteractiveParameterBuilder.Create(_harness.ClientApplication,
             _harness.ValidateInteractiveParameters(
                 expectedLoginHint: MsalTestConstants.DisplayableId);
         }
+
+        [TestMethod]
```

Comment:
The SeleniumWebUi from the Integration tests is a prime candidate to use this extensibily point. It currently implements IWebUi but injects itself via a factory. Should be very easy to replace with this new call.